phase1/src/lib.rs

```diff
@@ -6,15 +6,17 @@ pub mod objects;
 pub use objects::*;
 
 mod computation;
+mod initialization;
 mod key_generation;
+mod serialization;
```

Comment:
Can you confirm that serialization is being used in WASM during runtime and not merely for testing purposes?

Reply:
It is used for testing purpose for now.